werkende_callgraph_functie.py

```diff
@@ -0,0 +1,117 @@
+from pycallgraph2 import PyCallGraph
```

Comment:
Wat is deze file?

Reply:
ik was hier even aan het knutselen met die callgraphs. ik wilde niet dat de app.py vervuild raakte vandaar de kopie